src/main/java/edu/hm/hafner/analysis/parser/Armcc5CompilerParser.java

```diff
@@ -19,7 +19,7 @@ public class Armcc5CompilerParser extends LookaheadParser {
     @Serial
     private static final long serialVersionUID = -2677728927938443701L;
 
-    private static final String ARMCC5_WARNING_PATTERN = "^(.+)\\((\\d+)\\): (warning|error):  #(.+): (.+)$";
+    private static final String ARMCC5_WARNING_PATTERN = "^(?:\"(.+)\", line (\\d+): (warning|error) #(.+): (.+)|(.+)\\((\\d+)\\): (warning|error):\\s+#(.+): (.+))$";
```

Comment:
Can you please used named groups? (Example https://jdriven.com/blog/2020/04/Java-Joy-Using-Named-Capturing-Groups-In-Regular-Expressions)